.travis.yml

```diff
@@ -51,8 +51,8 @@ before_install:
 - cp ../test/google_credentials.json google_credentials.json
 - cd -
 # Configuration files
-- cp test/config.js.example config.js
-- cp test/config.js.example test/config.js
+- cp config.js.example config.js
+- cp config.js.example test/config.js
```

Comment:
Does it mean that we don't need the `example` files inside the `test/` directory? If so, can we delete them?

Reply:
Hmm, we still do need `test/config.yml.example` and we might still use `test/config.js.example` if we're not running the nginx proxies locally, which I haven't made autostart for the local test env. It gets started automatically for Travis jobs.

Reply:
OK sounds good!